packages/next-tinacms-s3/src/handlers.ts

```diff
@@ -3,21 +3,21 @@
 */
 
 import {
-  _Object,
-  S3Client,
-  S3ClientConfig,
+  DeleteObjectCommand,
+  DeleteObjectCommandInput,
   ListObjectsCommand,
   ListObjectsCommandInput,
   PutObjectCommand,
   PutObjectCommandInput,
-  DeleteObjectCommand,
-  DeleteObjectCommandInput,
+  S3Client,
+  S3ClientConfig,
+  _Object,
 } from '@aws-sdk/client-s3'
-import { Media, MediaListOptions } from 'tinacms'
-import path from 'path'
 import fs from 'fs'
-import { NextApiRequest, NextApiResponse } from 'next'
 import multer from 'multer'
+import { NextApiRequest, NextApiResponse } from 'next'
+import path from 'path'
+import { Media, MediaListOptions } from 'tinacms'
 import { promisify } from 'util'
```

Comment:
This looks like just re-ordering - if that's the case, please revert these changes